Fixed: Header logo displays correctly according to device screen size

qKa opened the pull request:
- Adjusted the behavior of the header to ensure it displays the correct logo size based on the device's screen size.

mentor-tara reviewed (approved):
Go go go :smile:[](prolog-3)

qKa committed:
Fixed: Header logo displays correctly according to device screen size

- Adjusted the behavior of the header to ensure it displays the correct logo size based on the device's screen size.

mentor-tara reviewed (approved):
Go go go :smile:[](prolog-3)
